Kiss.Bff/Extern/Klantinteracties/PostContactmomentenCustomProxy.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Net.Http.Headers;
+using System.Text.Json.Nodes;
+using System.Text.Json;
+using Kiss.Bff.Afdelingen;
+using Microsoft.AspNetCore.Mvc;
+using System.Security.Claims;
+
+namespace Kiss.Bff.Extern.Klantinteracties
+{
+    [Route("api")]
+    [ApiController]
+    public class PostContactVerzoek : ControllerBase
+    {
+        private readonly GetMedewerkerIdentificatie _getMedewerkerIdentificatie;
+        private readonly HttpClient _httpClient;
+        private readonly KlantinteractiesProxyConfig _klantinteractiesProxyConfig;
+
+        public PostContactVerzoek(GetMedewerkerIdentificatie getMedewerkerIdentificatie, HttpClient httpClient, KlantinteractiesProxyConfig klantinteractiesProxyConfig)
+        {
+            _getMedewerkerIdentificatie = getMedewerkerIdentificatie;
+            _httpClient = httpClient;
+            _klantinteractiesProxyConfig = klantinteractiesProxyConfig;
+        }
+
+        [HttpPost("postinternetaak")]
+        public IActionResult PostInterneTaak([FromBody] JsonObject parsedModel)
```

Comment:
stop deze functie maar in PostKlantContactenCustomProxy.cs, dan kan je deze class opheffen. 
(naamgeving klopt nu niet en hij past prima bij PostKlantContacten, daar zit alles in wat er voor contactmomenten en contactverzoeken geproxied moet worden)

Reply:
gedaan